src/toast.rs

```diff
@@ -59,6 +60,7 @@ pub struct Toast {
     pub(crate) height: f32,
     pub(crate) width: f32,
     pub(crate) closable: bool,
+    pub (crate) show_progress_bar: bool,
```

Comment:
Please format your code by removing whitespace between `pub` and `(crate)`, Running `cargo fmt` should automatically fix this.